src/data/contributors/ragul.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Ragul M",
+    "skills": ["JAVA", "SQL", "HTML/CSS/JAVASCRIPT"],
+    "cause": "community",
+    "image": "Avatar6",
```

Comment:
This is a girl avatar:
![image](https://user-images.githubusercontent.com/48255244/103457846-9b08ae00-4d28-11eb-943c-57849ec0f1d1.png)
You could choose your avatar from : https://github.com/codeforcauseorg/pledge/tree/master/static/images/avatar